mirgecom/boundary.py

```diff
@@ -1848,7 +1850,7 @@ def grad_temperature_bc(self, dcoll, dd_bdry, grad_t_minus, normal, **kwargs):
         return self._adiabatic.grad_temperature_bc(grad_t_minus, normal)
 
 
-class LinearizedOutflowBoundary(PrescribedFluidBoundary):
+class LinearizedOutflowBoundary(MengaldoBoundaryCondition):
```

Comment:
I don't recall seeing this boundary type in Mengaldo, and I think maybe if we are going to hard-code 2D boundaries, maybe they should have `2D` in the name of the boundary class.

Reply:
Yes, they are not described in there. But I want to apply the BCs as the paper recommends without duplicating the existing code.